### PR TITLE
ZNTA-758 Glossary Usability

### PR DESCRIPTION
**Pull request by lukebrooker**
Should fix these problems:

- Add search, loading and reset icons to filter input
- Make it more obvious to select a language
- Grey out info icons if they have no information
- Add empty state if there are no terms available
- Add tooltip to show entire term descriptions
- Column heading for "translations" should be the name of the language being translated
- Make modal cover entire screen (including header)

From https://zanata.atlassian.net/browse/ZNTA-758

### CODE REVIEW

**Comment by lukebrooker** [comment]
@aeng It should be up to date now.

Do you think you could hook up the filter loading properly?

File: zanata-war/src/main/webapp/WEB-INF/template/template_ui.xhtml [hunk 1]
```diff
@@ -32,6 +32,8 @@
     <link type="text/css" rel="stylesheet" href="#{assets['css/style.min.css']}"/>
     <!--<![endif]-->
 
+    <link type="text/css" rel="stylesheet" href="#{request.contextPath}/bundle.css"/>
```
Comment (aeng) [inline zanata-war/src/main/webapp/WEB-INF/template/template_ui.xhtml]: Need to check if this actually works

**Comment by aeng** [comment]
Apart from checking the css import is working, everything else if fine. :+1:

**Comment by lukebrooker** [comment]
@aeng the CSS was working fine for me. Did you want to test it yourself?

**Comment by definite** [comment]
To test:
-[ ] Add search, loading and reset icons to filter input
-[ ] Make it more obvious to select a language
-[ ] Grey out info icons if they have no information
-[ ] Add empty state if there are no terms available
-[ ] Add tooltip to show entire term descriptions
-[ ] Column heading for "translations" should be the name of the language being translated
-[ ] Make modal cover entire screen (including header)

**Comment by aeng** [comment]
@lukebrooker the css import works fine. Passing this to QA.

**Comment by definite** [comment]
The translations is still not the name of language being translated.

**Comment by definite** [comment]
:+1:

**Pull request merged**